examples/javascript/button.js

```diff
@@ -26,7 +26,7 @@
 var groveSensor = require('jsupm_grove');
```

Comment:
Additionally, the [Intel website](https://software.intel.com/en-us/iot/hardware/sensors/grove-button) uses `jsupm_grove` too.